modules/images/webp-uploads/load.php

```diff
@@ -8,6 +8,39 @@
  * @package performance-lab
  */
 
+/**
+ * Can load function to determine if the module is already merged into WordPress core or loaded by
+ * another plugin.
```

Comment:
This function doesn't check whether the code is already loaded by another plugin, it only checks whether the code is already merged into WordPress core.
```suggestion
 * Determines whether the feature can be loaded, or whether is already merged into WordPress core.
 *
 * If it cannot be loaded, it will add an action to display a WordPress admin notice.
```